docs/user/rstudio/ide/guide/tools/version-control.qmd

```diff
@@ -54,43 +54,43 @@ An excellent resource for learning more about Subversion and how to use it is th
 
 ## Getting Started
 
-Once you've installed your preferred Version Control system, you'll need to activate it on your system by following these steps:
+Once a Version Control system has been installed, it needs to be activated on the system by following these steps:
 
 1.  Go to **Global Options** (from the Tools menu)
 
-2.  Click **Git/SVN**
+2.  Press **Git/SVN**
```

Comment:
If this is a button, we should change "Press" to Click.

Reply:
Changed to click